src/EFCore.Relational/Migrations/MigrationBuilder.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.EntityFrameworkCore.Migrations;
 
 /// <summary>
-///     A builder providing a fluentish API for building <see cref="MigrationOperation" />s.
+///     A builder providing a fluent-ish API for building <see cref="MigrationOperation" />s.
```

Comment:
"ish" is maybe a bit too spoken language for this context... Or just say fluent?

```suggestion
///     A builder providing a fluent-like API for building <see cref="MigrationOperation" />s.
```